equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,134 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
-
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
-	}
-`, name)
-}
-
-func TestAccFabricReadServiceProfileByUuid(t *testing.T) {
+func TestAccFabricCreateServiceProfile_SP_FCR_ba(t *testing.T) {
+	spName1 := "fabric_tf_acc_test_01"
+	spName2 := "fabric_tf_acc_test_02"
+	typel2 := "L2_PROFILE"
+	portType := "XF_PORT"
+	ports := GetFabricEnvPorts(t)
+	portTcFirst := ports["pfcr"]["dot1q"][0]
```

Comment:
These port calls will sometimes cause an issue if the ENV variables aren't set. Like in the `Go Tests` GHA workflow for the repo. You'll need to wrap them like we did in connections acceptance tests.

```
ports := GetFabricEnvPorts(t)
connectionsTestData := GetFabricEnvConnectionTestData(t)
var publicSPName, portUuid string
if len(ports) > 0 && len(connectionsTestData) > 0 {
  publicSPName = connectionsTestData["pfcr"]["publicSPName"]
  portUuid = ports["pfcr"]["dot1q"][0].Uuid
}
```

This protects the tests and allows them to be skipped when the test environment doesn't have `TF_ACC` set.

Reply:
done